material-components/material-design-for-wordpress
# Fix material color palette with WP 5.7

## Summary


See #116

### Testing steps:

1. Add a button.
1. Go to color and background color.
1. Click color palette. See material color palette. 

<img width="518" alt="image" src="https://user-images.githubusercontent.com/5015489/117674708-8900b580-b1c9-11eb-92c6-f9399f21f161.png">


## Checklist

- [x] My pull request is addressing an [open issue](https://github.com/material-components/material-design-for-wordpress/issues) (please create one otherwise).
- [x] My code is tested and passes existing [tests](https://github.com/material-components/material-design-for-wordpress/blob/main/contributing/engineering.md#tests).
- [x] My code follows the [Contributing Guidelines](https://github.com/material-components/material-design-for-wordpress/blob/main/contributing.md) (updates are often made to the guidelines, check it out periodically).
